roles/mistral/tasks/config.yml

```diff
@@ -4,6 +4,14 @@
     src: mistral.conf
     dest: /etc/mistral/mistral.conf
 
+- name: Set mistral config values
+  sudo: true
+  ini_file:
+    dest=/etc/mistral/mistral.conf
```

Comment:
It's better to handle/template variables in this existing file `/etc/mistral/mistral.conf`, see `st2/templates` for example.